### ISSUE TITLE
Mitogen website out of date (release notes, documentation, etc)

### ISSUE THREAD

**Issue by intgr** [mitogen-hq/mitogen#962]
Looks like Mitogen release notes and official documentation hasn't been updated for ages: https://mitogen.networkgenomics.com/changelog.html still displays "v0.3.0 (unreleased)", even though 0.3.3 has been released months ago.

**Comment by jonathan-vb-cf** [comment]
this is really anoying to not knowing what has been changed

**Comment by moreati** [comment]
Sitrep and brain dump
- https://mitogen.networkgenomics.com is hosted in a Netlify account, built from an out of date branch
- Access to that account can't be delegated to members of mitogen-hq organisation
- With some code fixes (https://github.com/mitogen-hq/mitogen/commit/6aa4fd3573cf4a081017c19c4cc726fbc27d8853, https://github.com/mitogen-hq/mitogen/commit/63457b4866b5017b9ec4cd664d73aecad0bac5e7) I was able to generate https://spiffy-croissant-696d93.netlify.app/ in my own Netlify account. This is a PoC, throwaway URL.
- David is willing/able to repoint the DNS once that effort is ready
- I also tried using GitHub Pages instead (https://github.com/mitogen-hq/mitogen/commit/a744f4d4cfaf04fa60eed49b1014746b46806f31) currently visilble at https://moreati.github.io/mitogen/. This is a PoC, throwaway URL.

A new Netlify account is the more likely outcome. I don't think I can make GHP work without the /mitogen path component.

**Comment by moreati** [comment]
Fixes from PR #1026 have been merged into docs-master branch. Both https://spiffy-croissant-696d93.netlify.app/ (my Netlify account) and https://mitogen.networkgenomics.com/ (@dw's account) have rebuilt with it. The Mitogen website now reflects latest released (and unreleased) code.

**Comment by moreati** [comment]
https://mitogen.networkgenomics.com/ansible_detailed.html now points directly to release 0.3.4, and https://mitogen.networkgenomics.com/changelog.html shows recent changes.
